bullet_train/docs/super-scaffolding/sortable.md

```diff
@@ -1,5 +1,17 @@
 # Super Scaffolding with the `--sortable` option
```

Comment:
This is really new docs for the new sortable controller, but it looks like extensive changes to the old docs Because Git.